src/mapml/layers/MapMLLayer.js

```diff
@@ -909,12 +910,13 @@ export var MapMLLayer = L.Layer.extend({
               var varName = v[1],
                   inp = serverExtent.querySelector('map-input[name='+varName+'],map-select[name='+varName+']');
               if (inp) {
-
+                
                 if ((inp.hasAttribute("type") && inp.getAttribute("type")==="location") && 
-                    (!inp.hasAttribute("min" || !inp.hasAttribute("max"))) && 
```

Comment:
How did this even work?

Reply:
not sure so I just add a bracket for it lol, hope it works well